### DISCUSSION TITLE
Add manual sorting of Text Snippets to the "Editor Toolbar"

### DISCUSSION THREAD

**Discussion by irmod** [damionx7/Zettel-Notes-Documentation#174] [Feature Requests]
Could you add the ability to manually sort snippets.

You implemented such a sorting mechanism in the "Saved Searches" list.

For me, the main process in "Zettel Notes" is text editing.  That's why a convenient "Edit Toolbar" is so important.

I will be able to place the most frequently used snippets at the top of the list.  Editing a note will be more convenient and faster.

![Text_Snippets.png](https://github.com/user-attachments/assets/43e02bad-0e5d-444a-94a0-3e970e575370)

**Comment by irmod** [comment]
@damionx7, could you add a manual sorting option for snippets in the Editor Toolbar. I often use snippets when typing. I have a lot of snippets. To get to the desired snippet you have to scroll through the list several times and at the same time be extremely careful so as not to miss... The names of the snippets are short and many are similar...

The ideal option for me is if in the future you can make a modal window with a search like selecting tags to select snippets...

Thank you for your Zettel Notes

**Comment by damionx7** [comment]
Hey! I have added this feature. Please check out this video on how to insert text snippets while editing.


https://github.com/user-attachments/assets/d534d6cd-587a-435e-9dac-87c8aeba723e

**Reply by irmod** [reply]
Wow!  @damionx7, this is very cool!!!
